google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpc.java

```diff
@@ -56,11 +56,11 @@
 import com.google.spanner.admin.instance.v1.UpdateInstanceConfigMetadata;
 import com.google.spanner.admin.instance.v1.UpdateInstanceMetadata;
 import com.google.spanner.v1.*;
+import java.time.Duration;
```

Comment:
This may impact
```
 ServerStream<PartialResultSet> executeStreamingPartitionedDml(
      ExecuteSqlRequest request, @Nullable Map<Option, ?> options, Duration timeout);
 ```

Reply:
https://github.com/googleapis/java-spanner/blob/439e105d05107f7220ee2a19aeb55af4c074ee55/google-cloud-spanner/src/main/java/com/google/cloud/spanner/PartitionedDmlTransaction.java#L92 is the only production usage of method. Moreover, the interface is marked as `@InternalApi`. I relied on the tests to tell whether this may break something.

Reply:
@rahul2393 does it make sense to treat this code as internal?

Reply:
yes @diegomarquezp you can treat this as an internal class.

Reply:
Thank you